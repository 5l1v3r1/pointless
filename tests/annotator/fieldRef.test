
x = y.foo
y = {foo = Bar}