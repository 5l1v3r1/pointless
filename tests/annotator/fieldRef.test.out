
x = y.foo
access: y global: 1

y = {foo = Bar}
entry: foo index: 0

x = y.foo
entry: x index: 0

y = {foo = Bar}
entry: y index: 1
